src/main/java/org/terasology/potions/system/DrinkPotionAuthoritySystem.java

```diff
@@ -42,10 +42,7 @@
 import org.terasology.potions.PotionCommonEffects;
 import org.terasology.potions.component.PotionComponent;
 import org.terasology.potions.component.PotionEffect;
-import org.terasology.potions.effect.AlterationToHerbEffectWrapper;
-import org.terasology.potions.effect.DoNothingEffect;
-import org.terasology.potions.effect.HarmEffect;
-import org.terasology.potions.effect.HealEffect;
+import org.terasology.potions.effect.*;
```

Comment:
Please avoid wildcard imports like this. There's usually an IDE setting that needs to be set to not collapse into wildcard unless above some high number :-)